src/server/utilities/logResponseTime/index.js

```diff
@@ -12,9 +17,20 @@ const logResponseTime = (req, res, next) => {
 
   res.on('finish', () => {
     const elapsedHrTime = process.hrtime(startHrTime);
-    logger.info(SERVER_RESPONSE_TIME, {
+
+    const nanoseconds = elapsedHrTime[0] * NS_PER_SEC + elapsedHrTime[1];
+
+    let responseTimeLogger = logger.debug;
+    let LOG_EVENT = SERVER_RESPONSE_TIME;
+
+    if (nanoseconds > THREE_SECONDS_IN_NANOSECONDS) {
+      responseTimeLogger = logger.info;
```

Comment:
No big deal but should this maybe be warning?

Reply:
Yes, good shout!